bin/validate.js

```diff
@@ -36,11 +36,11 @@ const VALIDATE_PATH = path.resolve(__dirname, '../scripts/validation/validate.js
       .on('fail', () => { hasFailedTests = true; })
       .on('end', () => {
         if (hasFailedTests) {
-          process.exitCode = 1;
+          process.exit(1);
```

Comment:
> Thanks to the commit log, I understand this is because of Windows. I would love to have a short synchronous discussion to clarify what we're doing, because I have a growing feeling we don't really understand what's happening in this file 😉